inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/project/EvaluationSimulationPanel.java

```diff
@@ -0,0 +1,300 @@
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import de.agilecoders.wicket.webjars.request.resource.WebjarsCssResourceReference;
+import de.agilecoders.wicket.webjars.request.resource.WebjarsJavaScriptResourceReference;
+import de.tudarmstadt.ukp.clarin.webanno.api.DocumentService;
+import de.tudarmstadt.ukp.clarin.webanno.model.AnnotationDocument;
+import de.tudarmstadt.ukp.clarin.webanno.model.Project;
+import de.tudarmstadt.ukp.clarin.webanno.model.SourceDocument;
+import de.tudarmstadt.ukp.clarin.webanno.security.UserDao;
+import de.tudarmstadt.ukp.inception.recommendation.api.evaluation.IncrementalSplitter;
+import de.tudarmstadt.ukp.inception.recommendation.api.model.Recommender;
+import de.tudarmstadt.ukp.inception.recommendation.api.recommender.RecommendationEngine;
+import de.tudarmstadt.ukp.inception.recommendation.api.recommender.RecommendationException;
+import de.tudarmstadt.ukp.inception.recommendation.imls.external.ExternalRecommender;
+import de.tudarmstadt.ukp.inception.recommendation.imls.external.ExternalRecommenderFactory;
+import de.tudarmstadt.ukp.inception.recommendation.imls.external.ExternalRecommenderTraits;
+import de.tudarmstadt.ukp.inception.recommendation.imls.opennlp.doccat.OpenNlpDoccatRecommender;
+import de.tudarmstadt.ukp.inception.recommendation.imls.opennlp.doccat.OpenNlpDoccatRecommenderFactory;
+import de.tudarmstadt.ukp.inception.recommendation.imls.opennlp.doccat.OpenNlpDoccatRecommenderTraits;
+import de.tudarmstadt.ukp.inception.recommendation.imls.opennlp.ner.OpenNlpNerRecommender;
+import de.tudarmstadt.ukp.inception.recommendation.imls.opennlp.ner.OpenNlpNerRecommenderFactory;
+import de.tudarmstadt.ukp.inception.recommendation.imls.opennlp.ner.OpenNlpNerRecommenderTraits;
+import de.tudarmstadt.ukp.inception.recommendation.imls.opennlp.pos.OpenNlpPosRecommender;
+import de.tudarmstadt.ukp.inception.recommendation.imls.opennlp.pos.OpenNlpPosRecommenderFactory;
+import de.tudarmstadt.ukp.inception.recommendation.imls.opennlp.pos.OpenNlpPosRecommenderTraits;
+import de.tudarmstadt.ukp.inception.recommendation.imls.stringmatch.StringMatchingRecommender;
+import de.tudarmstadt.ukp.inception.recommendation.imls.stringmatch.StringMatchingRecommenderFactory;
+import de.tudarmstadt.ukp.inception.recommendation.imls.stringmatch.StringMatchingRecommenderTraits;
+
+public class EvaluationSimulationPanel
+    extends Panel
+{
+    private static final long serialVersionUID = 4306746527837380863L;
+
+    private static final String CHART_CONTAINER = "chart-container";
+    private static final String SIMULATION_START_BUTTON = "simulation-start-button";
+    
+    private static final double TRAIN_PERCENTAGE = 0.8;
+    private static final int INCREMENT = 250;
+    private static final int LOW_SAMPLE_THRESHOLD = 10;
+    
+    private static final String FORM = "form";
+
+    private static final Logger log = LoggerFactory.getLogger(EvaluationSimulationPanel.class);
```

Comment:
static final fields should be all capitals.

Reply:
done.